Components/DayCalendar.js

```diff
@@ -9,50 +12,251 @@ var {
   ScrollView,
   View,
   Text,
+  PanResponder,
+  Image,
 } = React;
 
+type Event = {
+  title: string;
+  startAt: Date;
+  length: number;
+};
```

Comment:
Oh cool, so what's going on here? Is this part of RN? Is this a feature of JS that i've just been oblivious to? Is it a library?